### ISSUE TITLE
Flow type checking fails - Cannot use InAppPurchase as a type because it is an any-typed value.

### ISSUE THREAD

**Issue by rimzici** [dooboolab/react-native-iap#1341]
With RN v0.64.1 and flow-bin v^0.137.0, the flow fails with the following error

`Cannot use InAppPurchase as a type because it is an any-typed value. Type InAppPurchase properly, so it
is no longer any-typed, to use it as an annotation. [value-as-type]`

I am importing as follows

```javascript
import {
	purchaseErrorListener,
	purchaseUpdatedListener,
	type ProductPurchase,
	type PurchaseError,
	type InAppPurchase,
	type SubscriptionPurchase,
} from 'react-native-iap';
```

Used to work fine with "flow-bin": "^0.122.0" and "react-native": "0.63.4"

### Version of react-native-iap

"react-native-iap": "^6.0.6"

### Version of react-native

"react-native": "0.64.1"

### Expected behavior

flow should pass

### Actual behavior

flow type checking fails

### Steps to reproduce the behavior

Configure and run flow

**Comment by hyochan** [comment]
Honestly, I don't know much about flow. I am just relying on [flowgen](https://github.com/joarwilk/flowgen) for flow typing. Could you open up issue there too?

**Comment by hyochan** [comment]
@rimzici Looks like https://github.com/dooboolab/react-native-iap/pull/1377 was causing a problem.

This fix is released in 6.2.1. Please come back if you still face the problem.
